2_deploy_ses_aio/roles/setup_ses_configs/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+
+ses_config_path: "/tmp/ses_config"
```

Comment:
Can we omit this part and generate the file directly under `~/suse-osh-deploy` ?

Reply:
We should do this.

Reply:
I agree. To get something working  quick I basically just checkout the old playbook and copied the parts that were missing. I'll rework it to get it to a saner state.

Reply:
Done